build/Dockerfile.pluginbuilder

```diff
@@ -0,0 +1,37 @@
+## Copyright 2018 The Nakama Authors
```

Comment:
I'd suggest calling this `pluginbuilder.Dockerfile` as editors can highlight the file properly.

As a better alternative, and what the community convention is, create a folder called `pluginbuilder` and put this file as simply `Dockerfile` there.

Reply:
https://github.com/docker-library/golang/tree/ed78459fac108dab72556146b759516cc65ee109/1.11 or https://stackoverflow.com/questions/26077543/how-to-name-dockerfiles